docs/sdk/api/apiRequestBuilder.md

```diff
@@ -14,27 +14,31 @@ npm install --save @commercetools/api-request-builder
 <script>// global: CommercetoolsApiRequestBuilder</script>
 ```
 
-## `createRequestBuilder(customServices)`
+## `createRequestBuilder(projectKey, customServices)`
```

Comment:
Do you mean to refactor `createRequestBuilder()` to take in one object as argument containing projectKey and customServices? 😶 🙄